### PR TITLE
Observe cmd init

### PR DESCRIPTION
**Pull request by nthnieljson**

### CODE REVIEW

File: observe/observe.go [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+	Operation     string
+	Namespace     string
+	AllNamespace  bool
+	Labels        string
+	ShowLabels    bool
+	Since         string
+	CustomColumns string
+}
+
+// Get observability data
```
Comment (seswarrajan) [inline observe/observe.go:30]: Fix linter error

File: observe/observe.go [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+	proto "github.com/kubearmor/koach/protobuf"
+	"github.com/rodaine/table"
+	"google.golang.org/grpc"
+)
+
+// Options Structure
+type Options struct {
+	Operation     string
+	Namespace     string
+	AllNamespace  bool
```
Comment (seswarrajan) [inline observe/observe.go:23]: If above option(Namespace) is null/empty, can we cover all namespaces ?

**Review by seswarrajan — CHANGES_REQUESTED** [review]

File: observe/observe.go (hunk 2 above)
Comment (nthnieljson) [reply observe/observe.go:23]: no.

the behaviour is the same kubectl, if it is empty, then we will assume it is "default" namespace, I have provided "-A" flag to cover all namespaces

**Comment by Ankurk99** [comment]
Hey @nthnieljson! Can you please handle this PR?

**Review by daemon1024 — APPROVED** [review]

**Comment by daemon1024** [comment]
Ref https://github.com/kubearmor/koach/pull/7

**Comment by yasin-cs-ko-ak** [comment]
@nthnieljson Kindly resolve this conflict and also handle `go-sec` CI.
cc: @daemon1024 @seswarrajan

**Pull request closed without merging**